tests/tests.py

```diff
@@ -56,7 +56,7 @@ def stop(self):
 
 
 def test_suite():
-    docker_layer = DockerLayer('docker', 'crate/crate:test')
+    docker_layer = DockerLayer('docker', 'crate/crate:4.0.10')
```

Comment:
Why did this have to change?

Reply:
We do not have a `test` tagged image in https://registry.hub.docker.com/_/crate/  (should be made available when a release is advertised in the testing channels ?).  We are "testing" 4.0.10

Reply:
Maybe I'm missing something here. But isn't part of the purpose of these tests to verify that the `Dockerfile` builds an image that works correctly **before** we publish the images?

Using a final **released** image imho defeats the purpose of these tests. No?
I think what's missing here in this / the readme is that the image is being build?

Reply:
the tag that we were trying to pull/use (crate/crate:test) does not exist, for it to exist it must be created at some point, and at some other point it will need to be deleted and created again to point at the next image that must match the `test` tag. Or we simply edit the script to point to a tag that actually exists, such as 4.0.10 in this case.

Reply:
Previously the `test` tag was built locally:

https://github.com/crate/docker-crate/blob/b65ed56a865a5f7826bf1eacc5da3fd8a398ee90/tests/tests.py#L43-L49

https://github.com/crate/docker-crate/blob/b65ed56a865a5f7826bf1eacc5da3fd8a398ee90/tests/tests.py#L59

This is no longer working because the layer usage ( `suite.layer = docker_layer`) is a zope-testrunner specific thing and doesn't work that way under unittest. That'll have to be adapted.

Or we roll-back and keep using zope-testrunner (It can also be installed using virtualenv - there is no need to use buildout for that)